django/otto/templates/components/user_cost.html

```diff
@@ -0,0 +1,22 @@
+<div id="cost-bar-outer"
+     hx-get="{% url 'user_cost' %}"
+     hx-swap="outerHTML"
+     hx-trigger='load delay:60000ms'
```

Comment:
Is there a need to reload costs every 60 seconds?

Reply:
Kind of? I am OK with disabling the polling. I thought you might have an opinion on this.

Rationale for polling: not everything that incurs costs will automatically update the cost widget.
Specifically, actions in the Librarian modal (such as uploading/re-processing documents) don't update the cost widget. 
 There is also the possibility someone has multiple Otto tabs open. And then there are apps other than the AI assistant which can incur costs.

Reply:
No all good, thanks for the info